Resources/Private/Php/eID/JsonServer.php

```diff
@@ -1,7 +1,63 @@
 <?php
 
+namespace HDNET\Autoloader\eID\JsonServer;
```

Comment:
Please migrate this changes to your JsonServer object. No "functions" without class/namespace...